components/atom/table/src/index.js

```diff
@@ -78,19 +95,45 @@ const AtomTable = ({head, body, foot, fullWidth}) => {
 AtomTable.displayName = 'AtomTable'
 
 AtomTable.propTypes = {
+  /**
+   * Table head content
+   */
   head: PropTypes.array,
+  /**
+   * Table body content.
+   * You can define per row:
+   *  - colspan: as a number
+   *  - content: as a string or React component
+   *  - type: of cell (th,td)
+   *  - isNowrap: to add no-wrap behavior to the cell
+   */
   body: PropTypes.arrayOf(
     PropTypes.arrayOf(
       PropTypes.shape({
+        colspan: PropTypes.number,
         content: PropTypes.string.isRequired,
         type: PropTypes.oneOf(Object.values(CELL_TYPE)),
         isNowrap: PropTypes.bool
       })
     )
   ).isRequired,
+  /**
+   * Cell padding size (xs,x,m,l,xl)
+   */
+  cellPadding: PropTypes.oneOf(Object.values(CELL_PADDING)),
+  /**
+   * Table foot conntent
+   */
   foot: PropTypes.array,
-  fullWidth: PropTypes.bool
+  /**
+   * With fullWith you have a full width behavior
+   */
+  fullWidth: PropTypes.bool,
+  /**
+   * Add a default border bootom to all cells
+   */
+  borderBottom: PropTypes.bool
 }
 
-export {CELL_TYPE as AtomTableTypes}
+export {CELL_TYPE as AtomTableCellTypes, CELL_PADDING as AtomTableCellPadding}
```

Comment:
Given this const are not a constructor, I think that should be in camelCase instead PascalCase:

```suggestion
export {CELL_TYPE as atomTableCellTypes, CELL_PADDING as atomTableCellPadding}
```